### PR TITLE
Basic partner integration for org management

### PR DESCRIPTION
**Pull request by dpnova**
This reverts commit 5b56f26def85fa592e1d3ba77dbfc7c90e2b757a.

### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] If this change fixes a bug (or a performance problem), is a regression
  test (or a benchmark) included?
- [ ] Is a documentation update included (if this change modifies
  existing APIs, or introduces new ones)?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### CODE REVIEW

File: web/partners/tests/factories.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+from organisations.tests.factories import OrganisationFactory
+
+
+class PartnerFactory(factory.django.DjangoModelFactory):
+
+    class Meta:
+        model = PartnerAccount
+
+
+class ParterOrganisationFactory(factory.django.DjangoModelFactory):
```
Comment (ejjick) [inline web/partners/tests/factories.py]: misspelled

File: web/run_tests.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ pip install -r requirements_test.txt
 # only run tests if flake8 succeeds.
 export DJANGO_SECRET_KEY=1234abcd
 export DJANGO_SETTINGS_MODULE=web.settings.test
-flake8 --exclude=migrations && pytest \
+flake8 --exclude=migrations && pytest $1\
```
Comment (ejjick) [inline web/run_tests.sh]: maybe add a comment what the arg is used for

File: web/se/schema.py [hunk 3]
```diff
@@ -365,92 +373,102 @@ class Arguments:
...
+                address=shareholder.address,
+                sharetype_id=SharetypeModel.objects.filter(
+                    symbol=shareholder.share_type,
+                    organisation_id=organisation).first().id,
+                payment_status=shareholder.payment_status,
+                shares=shareholder.shares,
+                email=shareholder.email,
+                type=shareholder.type,
+                company_identifier_type=shareholder
+                .company_identifier_type,
```
Comment (ejjick) [inline web/se/schema.py]: line break not necessary

File: web/user_accounts/migrations/0011_auto_20181114_0346.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.0.9 on 2018-11-14 03:46
```
Comment (ejjick) [inline web/user_accounts/migrations/0011_auto_20181114_0346.py]: Not sure _where_ these changes come from. No corresponding `models.py` file.

Comment (dpnova) [reply web/user_accounts/migrations/0011_auto_20181114_0346.py]: They are from help_text changes. I'd rather just run with what migrations wants to do... don't wanna fight it.

**Pull request merged**